crypto/secp256k1/secp256k1_test.go

```diff
@@ -24,8 +24,8 @@ type keyData struct {
 var secpDataTable = []keyData{
 	{
 		priv: "a96e62ed3955e65be32703f12d87b6b5cf26039ecfa948dc5107a495418e5330",
-		pub:  "02950e1cdfcb133d6024109fd489f734eeb4502418e538c28481f22bce276f248c",
```

Comment:
This code only deals with uncompressed (our future code in CometBFT will have both, but here it's not possible since the compressed key type identifier was reused in the old peppermint)

Reply:
Resolved